src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    private Random random = new Random();
+    private ColorSupplier supplier = new ColorSupplier();
+    private static final int NUMBER_LIMIT = 100;

```

Comment:
Java CI failed due to the wrong order of this constant declaration, refer [here](https://mate-academy.github.io/style-guides/java/java.html#s3.4.2-ordering-class-contents). Please, declare it before random and supplier instances. You could also add an empty line after its declaration :)